### DISCUSSION TITLE
Shader error messages

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre-gl-js#3682] [Q&A]
I was editing some shaders yesterday and of course made some typos and bugs. To my surprise, there were no error messages displayed in the chrome browser console. Also the syntax highlighting in vscode did not show any errors in the .glsl files.

Is there a way to get some error messages/debug info when writing shaders?

**Comment by HarelM** [comment]
Might be related to this PR:
- https://github.com/maplibre/maplibre-gl-js/pull/2655

I'm not very familiar with this part.
cc: @birkskyum.
